pants_plugins/sendwave/pants_docker/BUILD

```diff
@@ -1,19 +1,24 @@
-python_library(
+python_sources(
     name="pants_docker_library",
-    dependencies=["pants_plugins:pants",],
+    dependencies=[
+        "pants_plugins:pants",
+    ],
```

Comment:
Some of this diff is just autoformatting that happened when I ran `update-build-files` during this upgrade: https://www.pantsbuild.org/docs/create-initial-build-files#formatting-build-files-with-black Maybe we should follow the docs advice at some point and add this step to CI?

Reply:
You can also disable Black formatting if you want by setting `[update-build-files].fmt = false`.